src/dvclive/utils.py

```diff
@@ -1,12 +1,31 @@
+from __future__ import annotations
 import csv
 import json
 import os
 import re
 import shutil
-import webbrowser
 from pathlib import Path
 from platform import uname
-from typing import Union
+from typing import Union, List, Dict, TYPE_CHECKING
+import webbrowser
+
+if TYPE_CHECKING:
+    import numpy as np
+    import pandas as pd
+else:
+    try:
+        import pandas as pd
+    except ImportError:
+        pd = None
+
+    try:
+        import numpy as np
+    except ImportError:
+        np = None
+
+from .error import (
+    InvalidDataTypeError,
+)
```

Comment:
Please move this before `if TYPE_CHECKING` block. Also, adding a comma at the end hangs this, so you can remove it and keep it in a single line.
```suggestion
from .error import InvalidDataTypeError
```